.changeset/eight-pianos-divide.md

```diff
@@ -0,0 +1,24 @@
+---
+'@xstate/graph': major
```

Comment:
i'm not sure how those major bumps will behave here since this package is already in the prerelease mode (sort of). We can review what Changesets will do to all of this after this PR lands - then we'll be able to just tweak the content or the changeset files as needed